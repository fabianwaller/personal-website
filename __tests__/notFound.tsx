import PageNotFound from "@/app/not-found";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Not found", () => {
  it("renders 404 page unchanged", () => {
    const component = render(<PageNotFound />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
