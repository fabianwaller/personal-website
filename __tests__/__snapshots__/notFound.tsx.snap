// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found renders 404 page unchanged 1`] = `
<DocumentFragment>
  <div
    class="container relative ml-auto mr-auto w-screen max-w-custom px-8 md:px-0 flex flex-col items-center justify-center h-page-minus-header"
  >
    <div
      class="flex flex-col items-center justify-center gap-6"
    >
      <div
        class="flex flex-col items-center justify-center gap-4 text-center"
      >
        <h1>
          Page Not Found
        </h1>
        <span>
          The page you are looking for does not exist.
        </span>
      </div>
      <a
        href="/"
      >
        <button
          class="relative inline-flex items-center justify-center gap-1 whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 h-10 px-4 py-2 bg-primary text-title-normal hover:bg-primary-foreground"
        >
          Bring me back
        </button>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
